src/app/screens/Login/index.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import { isInvalid } from 'redux-form';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
+import ReactRouterPropTypes from 'react-router-prop-types';
+import PropTypes from 'prop-types';
+
+import loginActions from '../../redux/Login/actions';
+
+import Login from './layout';
+
+class LoginContainer extends Component {
+  handleSubmit = event => {
+    this.props.dispatch(loginActions.login(event));
+    this.props.history.push('/game');
```

Comment:
define a constants file with all the routes. 

```
export const ROUTES = {
   GAME: '/game'
}
```

Also, you should dispatch the `push` in `loginSuccess`